src/widgets/bookmark/BookmarkWidgetTile.tsx

```diff
@@ -39,6 +40,7 @@ interface BookmarkItem {
   href: string;
   iconUrl: string;
   openNewTab: boolean;
+  hideLink: boolean;
```

Comment:
Rename to ``hideHostname`` or ``hideBasename``. Link is the whole sequence - we only display a part of it.